### DISCUSSION TITLE
Lavavents cant hurt people "inside" submarines

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13190] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Volcanoes cant hurt people inside submarines even if they are in the trigger range

Causes the "volcano softlock": 
Sitting on top of volcanoes damages the submarine enough to not be able to repair it, but volcanoes dont actively kill the crew. The crew tries to repair the submarine, in vain. The only way out is restarting round.

It should either kill the crew fast or allow for repairs

Inside (no burn damage)
![image](https://github.com/Regalis11/Barotrauma/assets/73229309/c12d52b9-f6d3-49b4-9692-7c3629dd1487)

Outside (burn damage)
![image](https://github.com/Regalis11/Barotrauma/assets/73229309/2dfa4e08-67b0-4aeb-9305-bbecc05a8180)

### Version

v1.2.6.0 (Winter Update)

**Comment by SomeRandomNoobKekeke** [comment]
I thought geysers are pushing submarine up allowing you to repair engine and move it aside
